src/main/java/uk/gov/pay/webhooks/app/WebhooksModule.java

```diff
@@ -85,14 +82,24 @@ public CloseableHttpClient httpClient() {
                 .setSocketTimeout(timeoutInMillis)
                 .setCookieSpec(CookieSpecs.STANDARD)
                 .build();
-
+        
         var sslsf = new SSLConnectionSocketFactory(
                 SSLContexts.createDefault(),
                 new String[] { "TLSv1.2", "TLSv1.3" },
                 null,
                 SSLConnectionSocketFactory.getDefaultHostnameVerifier()
         );
-
+        
+        IdleConnectionMonitorThread staleMonitor
+                = new IdleConnectionMonitorThread(poolingConnManager, configuration.getWebhookMessageSendingQueueProcessorConfig().getConnectionPoolIdleConnectionTimeToLive().toSeconds());
+        staleMonitor.start();
+        try {
+            staleMonitor.join(1000);
```

Comment:
I don't understand why a join is required. Can the commit message be amended to explain how this is expected to work or add a comment if appropriate?